components/multi_page/view.coffee

```diff
@@ -10,11 +10,11 @@ module.exports = class MultiPageView extends Backbone.View
   events:
     'click .js-page': 'change'
 
-  initialize: ({ @title, @description, @pages }) ->
+  initialize: ({ @title, @description, @pages, @defaultPage }) ->
     @collection = new Backbone.Collection _.map @pages, (id, title) ->
       new Page id: id, title: title
 
-    @state = new Backbone.Model active: @collection.first().id
+    @state = new Backbone.Model active: if @defaultPage then @defaultPage else @collection.first().id
```

Comment:
Yeah `defaultPageId` makes more sense, and yes agreed it should probably look for the item in the collection. However if you are passing in the id, its pretty simple:
```coffeescript
@collection.get(@defaultPageId)
````